assets/components/adFreeSection/AdFreeSection.jsx

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+// ----- Imports ----- //
+import React from 'react';
+import GridImage from 'components/gridImage/gridImage';
+import LeftMarginSection from 'components/leftMarginSection/leftMarginSection';
+
+export default function AdFreeSection() {
+
+  return (
+    <LeftMarginSection modifierClasses={['blue']}>
+      <div className="component-ad-free">
+        <div className="component-ad-free__content">
+          <h3 className="component-ad-free__badge">New</h3>
+          <div className="component-ad-free__wrapper">
+            <div className="component-ad-free__col">
+              <h2 className="component-ad-free__header">Ad-free on all your devices</h2>
+              <p className="component-ad-free__copy">
+                {'Avoid the adverts and read without interruptions when you\'re signed in on your apps and theguardian.com'}
```

Comment:
Not sure about this? Eslint wanted me to escape the `'` character as an html entity if I put it as text, but that sounds like a readability nightmare

Reply:
I think that's ok, we do that a lot. It's `&#39;`?